utils/file_utils.py

```diff
@@ -54,20 +55,46 @@ def copy_target_into(target, into):
             shutil.copytree(target, into)   # copy directory
         else:
             shutil.copyfile(target, into)   # copy single file
+        if verbose:
+            print(' - Copy completed: from %s into %s' % (target, into))
     except (AssertionError, RuntimeError, EnvironmentError) as err:
         raise RuntimeError('Couldn\'t copy "%s" into "%s": %s' % (
             target, into, str(err)))
 
 
+def move_target(target, into, verbose=False):
```

Comment:
In normal English usage, "target" is the destination, not the source.  And "into" looks too much like a keyword.  Switch to "move_target(dest, src)" and use logging instead of "verbose"

Reply:
its a different issue for different merge.

Reply:
Bringing all the EXISTING routines into compliance can be done in a future merge.

This is a NEW routine, it should be done now while all of its invocations are still fresh in your head.

Reply:
It is not new. Its been there since the beginning of the script. I just added "verbose" to it.